### ISSUE TITLE
Tickets updated or closed posting in wrong discord channel.

### ISSUE THREAD

**Issue by cybergamezone** [gameserverapp/Platform#817]
I have been unable to figured out how or why when a report is closed or updated, it posts in the wrong channel. The channel name says "Tier-sub". But in the setting of the reports, and automation that are both in a different channel called "support" and that is correct. Screenshots attached.

https://cdn.discordapp.com/attachments/1225199016074150008/1225199402881388544/Screenshot_2024-04-03_144348.png?ex=662042dc&is=660dcddc&hm=2f016940a91a7607f7d519175e8b3f1f40cc1a17f37a88fb30e7532f39888b23&

Posting in this channel called tier-Sub: which is where I do not want it to post. Posting next ss for settings...
https://cdn.discordapp.com/attachments/1225199016074150008/1225199570187976865/Screenshot_2024-04-03_144528.png?ex=66204304&is=660dce04&hm=c18a5046602944dd4239e7e1749dc252c27c90571bbd2e6792e1460df531025e&

And set correctly here:
https://cdn.discordapp.com/attachments/1225199016074150008/1225199810164817960/Screenshot_2024-04-03_144622.png?ex=6620433d&is=660dce3d&hm=52b9262443ae7465424f821609e2eed45e3b254ce92c73088a0a3ef9d04e9441&

Basically trying to figure out where its set to "Tier_sub".

![Screenshot 2024-04-03 150455](https://github.com/gameserverapp/Platform/assets/156719466/2f000002-8ea7-46ab-b471-29af73f9a6ec)
